includes/admin/upgrades/v3/class-data-migrator.php

```diff
@@ -498,6 +498,57 @@ public static function orders( $data = null ) {
 			$total   = $meta['_edd_payment_total'][0];
 		}
 
+		// Build up the order address data. Actual insertion happens later, but we need this now to figure out the tax rate.
+		$order_address_data = array(
+			'name'         => trim( $user_info['first_name'] . ' ' . $user_info['last_name'] ),
+			'address'      => isset( $user_info['address']['line1'] )   ? $user_info['address']['line1']   : '',
+			'address2'     => isset( $user_info['address']['line2'] )   ? $user_info['address']['line2']   : '',
+			'city'         => isset( $user_info['address']['city'] )    ? $user_info['address']['city']    : '',
+			'region'       => isset( $user_info['address']['state'] )   ? $user_info['address']['state']   : '',
+			'country'      => isset( $user_info['address']['country'] ) && array_key_exists( strtoupper( $user_info['address']['country'] ), edd_get_country_list() )
+				? $user_info['address']['country']
+				: '',
+			'postal_code'  => isset( $user_info['address']['zip'] )     ? $user_info['address']['zip']     : '',
+			'date_created' => $date_created_gmt,
+		);
+
+		// Remove empty data.
+		$order_address_data = array_filter( $order_address_data );
+
+		$tax_rate_id = null;
+		$tax_rate = isset( $meta['_edd_payment_tax_rate'][0] )
```

Comment:
These two lines need to be changed to `_edd_payment_tax` in order to get the tax amount--once that is set then the `$tax_rate_id` can be retrieved.

Reply:
Not sure I'm following this. This line is getting the tax rate percentage, why should I get the tax amount instead?

Reply:
Sorry, my mistake, you are right. But somehow I'm ending up with no tax rate object being retrieved so still digging on that. Scratch that suggestion because it was totally wrong!